targets/lantiq-xrx200_legacy

```diff
@@ -0,0 +1,30 @@
+packages {
+	'-ltq-vdsl-vr9-vectoring-fw-installer',
+	'-kmod-ltq-vdsl-vr9-mei',
+	'-kmod-ltq-vdsl-vr9',
+	'-kmod-ltq-atm-vr9',
+	'-kmod-ltq-ptm-vr9',
+	'-kmod-ltq-deu-vr9',
+	'-ltq-vdsl-vr9-app',
+	'-dsl-vrx200-firmware-xdsl-a',
+	'-dsl-vrx200-firmware-xdsl-b-patch',
+	'-ppp-mod-pppoa',
+	'-ltq-dsl-base',
+}
+
+-- TP-Link
+
+-- CAVEAT: These devices don't have a dedicated WAN port.
+--         All ethernet ports on the device are bridged and
+--         used as WAN ports. Clients connected to these
+--         ports will be connected to your private network.
+
+device('tp-link-td-w8970', 'tplink_tdw8970', {
+	factory = false,
+})
+
+device('tp-link-td-w8980', 'tplink_tdw8980', {
+	factory = false,
+	aliases = {'tp-link-td-w9980'},
+	broken = true, -- 5GHz unsupported
+})
```

Comment:
This also required a documentation change of the supported target assigments.

Reply:
@maurerle https://github.com/freifunk-gluon/gluon/blob/main/docs/user/supported_devices.rst